core/src/main/scala/Monitoring.scala

```diff
@@ -204,7 +204,18 @@ trait Monitoring {
             else Process.eval_(modifyActive(cluster, _ + source)) ++ received
           }
 
-          Task.delay(logErrors(Task.fork(receivedIdempotent.run)(defaultPool)).runAsync(_ => ()))
+          def attemptedTask:Unit =
+            Task.fork(receivedIdempotent.run)(defaultPool)
+              .attempt
+              .flatMap(
+                _.fold(
+                  e => Task.delay(log.error(s"source=$source\tcluster=$cluster\t${e.getMessage}")),
+                  Task.now
+                )
+              )
+              .runAsync(_ => ())
```

Comment:
I thought so too.  It was written as such, I was hesitant to change it.  

just so we're clear, we wouldn't need the attempt + flatMap, just move the _.fold to runAsync